lxd/containers.go

```diff
@@ -757,27 +757,20 @@ func (d *lxdContainer) applyConfig(config map[string]string) error {
 	}
 
 	if lxcConfig, ok := config["raw.lxc"]; ok {
-		for _, line := range strings.Split(lxcConfig, "\n") {
-			contents := strings.SplitN(line, "=", 2)
-			key := contents[0]
-			value := ""
-			if len(contents) > 1 {
-				value = contents[1]
-			}
-
-			key = strings.Trim(key, " ")
-			value = strings.Trim(value, " ")
-
-			// empty lines
-			if key == "" {
-				continue
-			}
+		f, err := ioutil.TempFile("", "lxd_config")
```

Comment:
On Wed, Mar 11, 2015 at 10:32:33AM -0700, St�phane Graber wrote:
> > -		for _, line := range strings.Split(lxcConfig, "\n") {
> > -			contents := strings.SplitN(line, "=", 2)
> > -			key := contents[0]
> > -			value := ""
> > -			if len(contents) > 1 {
> > -				value = contents[1]
> > -			}
> > -
> > -			key = strings.Trim(key, " ")
> > -			value = strings.Trim(value, " ")
> > -
> > -			// empty lines
> > -			if key == "" {
> > -				continue
> > -			}
> > +		f, err := ioutil.TempFile("", "lxd_config")
> 
> Looks to me like we've been using the trailing underscore everywhere until now:
> 
>     stgraber@dakara:~/data/code/lxc/lxd$ grep -r TempFile
>     lxc/config.go:	f, err := ioutil.TempFile("", "lxc_profile_")
>     lxc/image.go:		f, err := ioutil.TempFile("", "lxc_image_")
>     lxd/images.go:	f, err := ioutil.TempFile(dirname, "image_")

criu:~/packages/go/src/github.com/lxc/lxd master grep -ri tempdir *
lxd/migration/migrate.go: checkpointDir, err := ioutil.TempDir("", "lxd_migration")
lxd/migration/migrate.go: imagesDir, err := ioutil.TempDir("", "lxd_migration")
lxd/migration/rsync_test.go:  source, err := ioutil.TempDir("", "source")
lxd/migration/rsync_test.go:  sink, err := ioutil.TempDir("", "sink")

Not for TempDir, though.

> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/365/files#r26234931